Sources/StreamChat/Models/Attachments/ChatMessageFileAttachment.swift

```diff
@@ -4,8 +4,6 @@
 
 import Foundation
 
-public typealias ChatMessageFileAttachment = _ChatMessageAttachment<AttachmentFilePayload>
-
```

Comment:
Why this? 🤔 `_ChatMessageAttachment` is the "base" and this is the concrete implementation for `AttachmentFilePayload`. IMO it was in the right place.

Reply:
Hmm, I always use the typealias as jump-to-definition / search for the implementation. I was quite surprised that the `ChatMessageFileAttachment` is not where `_ChatMessageAttachment`. IMHO it's more convenient to have typealias always where the aliased type resides... But I am happy to discuss this :)

Reply:
The `AttachmentFilePayload` is what matters in this case. `_ChatMessageAttachment` is just a box for it. I really belive the original place was correct 👍

Reply:
Okay I put it back where it was before.